printer_mods/Boingomw/Wago_mount/README.md

```diff
@@ -0,0 +1,20 @@
+# Wago_Mounts
+Wago 221 Lever Nuts are really cool, and easy to use (and not inexpensive).  The 221-412 Lever Nut can be used to make your heater bed modular.  You can get them here: https://www.amazon.com/Lever-Nuts-Compact-Splicing-Connector-Contains/dp/B093LQSQXG/ref=psdc_6426215011_t3_B08W3QXN9B?th=1
+
+There are also very nice mounting brackets for them.  The one I based mine on was created by deepfriedheroin and is available here: https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/deepfriedheroin/v2_bed_wagos
```

Comment:
Please make this a relative link, should be

    [V2 Bed Wagos](../../deepfriedheroin/v2_bed_wagos)